### PR TITLE
Generator image and running tests in container

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

This Pull Request adds `generator-image` and running tests in a container.

- [x] Dockerfile from generator-image
- [x] Dockerfile.tests for running tests in container
- [x] tests are passing

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pytest
```
Comment (TomasTomecek) [inline requirements.txt]: don't we track requirements in setup.cfg?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+FROM registry.fedoraproject.org/fedora:30
+
+ENV LANG=en_US.UTF-8 \
+    PYTHONDONTWRITEBYTECODE=yes \
+    HOME=/tmp/packit-generator
+
+COPY requirements.txt requirements.sh files/packit-run.sh /tmp/packit-generator/
```
Comment (TomasTomecek) [inline Dockerfile]: why copy the files in when they are not used?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,18 @@
+FROM registry.fedoraproject.org/fedora:30
```
Comment (TomasTomecek) [inline Dockerfile]: could you please document purpose of this image?

Comment (phracek) [reply Dockerfile]: Sure, but what do you think? I am completely lost. What would you like to write about it?
We can, of course, use 29.

Comment (jpopelka) [reply Dockerfile]: Tomas, neither I do understand your comment

File: Dockerfile (hunk 2 above)
Comment (jpopelka) [reply Dockerfile]: We do, except `requirements.txt` which solved my suggestion.

File: Dockerfile (hunk 3 above)
Comment (TomasTomecek) [reply Dockerfile]: My point is: what is the purpose of the image this dockerfile is suppose to build: how and where is the image meant to be used.

Comment (phracek) [reply Dockerfile]: Look at the latest comment.

**Pull request merged**